webapp/src/pages/applicationForm/components/ApplicationForm.js

```diff
@@ -257,16 +264,20 @@ class Section extends React.Component {
         .slice()
         .sort((a, b) => a.question.order - b.question.order),
       hasValidated: false,
-      validationStale: false
+      validationStale: false,
+
     };
   }
 
+
+  // onchange
```

Comment:
These comments don't seem useful since they just contain the name of the function on the next line?